test/bugsnag_test.exs

```diff
@@ -1,13 +1,14 @@
 defmodule BugsnagTest do
   use ExUnit.Case
 
+  @tag capture_log: true
   test "it doesn't raise errors if you report garbage" do
     Bugsnag.report(Enum, %{ignore: :this_error_in_test})
   end
 
   test "it handles real errors" do
     try do
-      :foo = :bar
```

Comment:
this was issuing warning in compilation, and can be replaced by explicit raising exception